test/test_02_testing.py

```diff
@@ -0,0 +1,91 @@
+import os, subprocess
+import pytest, textwrap
+from . import utils
+
+
+def test(tmpdir):
+    project_dir = str(tmpdir)
+
+    # generate sample project with a test
+    utils.generate_project(
```

Comment:
FYI, there is an incomplete refactor here, I had in mind to migrate these all over to the TemplateProjectC version as seen in 01_basic